linkerd/protocol/http/src/test/scala/io/buoyant/linkerd/protocol/http/ErrorResponderTest.scala

```diff
@@ -10,22 +10,24 @@ import io.buoyant.router.RoutingFactory
 import io.buoyant.test.Awaits
 import java.net.URLEncoder
 import java.nio.charset.StandardCharsets.ISO_8859_1
+
+import io.buoyant.linkerd.protocol.http.FramingFilter.FramingException
 import org.scalatest.FunSuite
 
 class ErrorResponderTest extends FunSuite with Awaits {
 
   val svc = Service.mk[Request, Response] { _ =>
     Future.exception(RoutingFactory.UnknownDst(Request(), s"foo\nbar"))
   }
-  val stk = ErrorResponder.module.toStack(
+  val stk = ErrorResponder.serverModule.toStack(
```

Comment:
I'd like to rewrite these tests to run with both the `clientModule` and the `serverModule`, but I don't think it's terribly important.